### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by misalazeem**
Added test.md in .gitignore
Created this pull request. Will be posting this without merging.
Made Changes to the readme file according to the Microverse Template.

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @misalazeem , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- The markup has been created :+1: 
## Required Changes ♻️
- [REQUIRED] Kindly make sure your linter test is passing :+1: as per the project requirements
- [REQUIRED] I suggest adding a professional README flle to your project as provided in this [template](https://github.com/microverseinc/readme-template)  to make your project professional.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by misalazeem** [comment]
> # STATUS: CHANGES REQUIRED ♻️ :
> Hi @misalazeem ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!  [ 68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f74385950663341434b656f754c7a454976722f67697068792e676966 ](https://camo.githubusercontent.com/59fa9d95c18dbd5a8e47490a9e4b48393f569685410f3c78d4d83a9764fa1392/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f74385950663341434b656f754c7a454976722f67697068792e676966) [ ](https://camo.githubusercontent.com/59fa9d95c18dbd5a8e47490a9e4b48393f569685410f3c78d4d83a9764fa1392/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f74385950663341434b656f754c7a454976722f67697068792e676966)
> 
> ## Highlights 🎩 👍
> * The markup has been created 👍
> 
> ## Required Changes ♻️
> * [REQUIRED] Kindly make sure your linter test is passing 👍 as per the project requirements
> * [REQUIRED] I suggest adding a professional README flle to your project as provided in this [template](https://github.com/microverseinc/readme-template)  to make your project professional.
>   _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

So lighthouse linter error wont go away. The problem is it keeps saying this: "ChromeLauncher Waiting for browser"
Then  returns with the following error
"[0321/174605.416109:WARNING:sandbox_linux.cc(393)] InitializeSandbox() called with multiple threads in process gpu-process.
[0321/174605.[85](https://github.com/misalazeem/Hello-Microverse-Module-1/actions/runs/4481125459/jobs/7879904748?pr=3#step:5:86)5580:WARNING:bluez_dbus_manager.cc(247)] Floss manager not present, cannot set Floss enable/disable.

Unable to connect to Chrome

Error: Process completed with exit code 1.
##[debug]Finishing: Lighthouse Report"

Kindly guide.

File: README.md [hunk 1]
```diff
@@ -1 +1,228 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (ghost) [inline README.md:225]: - [ ] Good work making the previous changes although an `MIT` license is lacking in your project. Kindly add it to make your readme file more professional

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @misalazeem , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @misalazeem, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**